pages/exercises/index.js

```diff
@@ -55,15 +56,26 @@ export default function HomePage({ exercises, muscleGroups }) {
     setMuscles(muscleGroups);
   }
 
+  const { data: session } = useSession();
+
   return (
     <StyledSection>
       <HeadlineSection>
-        <H1>
-          WELCOME TO YOUR <br />
-          EXERCISE LIST
-        </H1>
+        {session ? (
+          <H1>
+            WELCOME TO YOUR <br />
+            EXERCISE LIST,{" "}
+            <Username>{session.user.name || session.user.username}</Username>!
```

Comment:
```suggestion
            <Username>{session.user.name}</Username>!
```
Why dealing with session.user.username? Per default you have the session.user.name what you can use.